### PR TITLE
WA-1632: implement BPAY lookup SDK

### PR DESCRIPTION
**Pull request by hailefrollo**

### CODE REVIEW

File: Sources/FrolloSDK/Model/Transient/Payments/VerifyBPAYResponse.swift [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    }
+    
+    /// The validity of the biller
+    public let valid: Bool
+    
+    /// The code of the biller
+    public let billerCode: String
+    
+    /// The name of the biller
+    public let billerName: String
```
Comment (sincychacko-frollo) [inline Sources/FrolloSDK/Model/Transient/Payments/VerifyBPAYResponse.swift:42]: This is an optional property. Same for crn, billerMinAmount & billerMaxAmount

Comment (sincychacko-frollo) [reply Sources/FrolloSDK/Model/Transient/Payments/VerifyBPAYResponse.swift:42]: It will be returned only if valid is true

Comment (sincychacko-frollo) [reply Sources/FrolloSDK/Model/Transient/Payments/VerifyBPAYResponse.swift:42]: Also, can you check in postman with an invalid biller code to check if billerCode is returned? If not make it optional as well & ask the Backend guys to update the doc if required?

**Review by sincychacko-frollo — APPROVED** [review]

File: Sources/FrolloSDK/Model/Transient/Payments/VerifyBPAYResponse.swift (hunk 1 above)
Comment (hailefrollo) [reply Sources/FrolloSDK/Model/Transient/Payments/VerifyBPAYResponse.swift]: @sincychacko-frollo Payam mentioned the optional values in this ticket https://frollo.atlassian.net/browse/WA-1631 👍

Comment (sincychacko-frollo) [reply Sources/FrolloSDK/Model/Transient/Payments/VerifyBPAYResponse.swift]: Awesome! Please update accordingly.

Comment (hailefrollo) [reply Sources/FrolloSDK/Model/Transient/Payments/VerifyBPAYResponse.swift]: @sincychacko-frollo yes, I updated my PR :)

**Review by dipeshfrollo — APPROVED** [review]

**Pull request merged**